generated/niscope/niscope.proto

```diff
@@ -1439,6 +1439,7 @@ message FetchArrayMeasurementRequest {
     ArrayMeasurement array_meas_function = 4;
     sint32 array_meas_function_raw = 5;
   }
+  sint32 meas_wfm_size = 6;
```

Comment:
Note: Didn't need any special metadata to get this param in the right spot which was nice so this shouldn't be a breaking change.